src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.ServiceModel.Duplex.builds" />
```

Comment:
When I include that I get...
`error : Path 'lib/netstandard1.1' was specified in PathsToInclude but was not found in the package System.ServiceModel.Duplex/4.3.0. [C:\OSS\WCF\src\System.ServiceModel.Duplex\pkg\System.ServiceModel.Duplex.pkgproj]`

Why is it looking under `4.3.0` and not `4.4.0-beta-25221-02`?

In my packages folder I also see that I have `packages\System.ServiceModel.Duplex\4.4.0-beta-25221-02\ref\netstandard1.1` but not `netstandard1.3`
and
`C:\OSS\WCF\packages\System.ServiceModel.Duplex\4.4.0-beta-25221-02\lib\netstandard1.3` but not `netstandard1.1
`Is this correct?